### PR TITLE
Update observation vital sign proprietory code with standard code

### PR DESCRIPTION
**Pull request by ab082977**
Description
----
1_observation
2_observation

Replaced Observation proprietary code with standard code
PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by BS062721 — APPROVED** [review]

**Review by fpivonka — APPROVED** [review]

File: lib/resources/r4/observation.yaml [hunk 1]
```diff
@@ -92,14 +92,12 @@ fields:
...
-            "display": 'Height/Length Measured',
-            "userSelected": true
+            "system": 'http://loinc.org',
+            "code": '8331-1'
           }
         ],
-        "text": 'Height/Length Measured'
-      }
+        "text": 'Temperature Oral'
+      },
```
Comment (BS062721) [inline lib/resources/r4/observation.yaml:100]: i think this is an extra coma?

**Review by lalitha94 — APPROVED** [review]

File: lib/resources/r4/observation.yaml (hunk 1 above)
Comment (ab082977) [reply lib/resources/r4/observation.yaml]: fix for this 
a0eb984[](https://github.com/cerner/fhir.cerner.com/pull/472/commits/a0eb984f468d92092fd31cd80ac6a6ddedd08bf6)

Comment (ab082977) [reply lib/resources/r4/observation.yaml]: obs_3

**Review by BS062721 — APPROVED** [review]

**Comment by lalitha94** [comment]
Verified on live
Screen Shot 2020-09-24 at 6 21 53 PM

**Pull request merged**